Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -0,0 +1,59 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import AwsCIo
+import struct Foundation.Date
+import struct Foundation.Data
+import class Foundation.FileHandle
+import AwsCCommon
+
+@inlinable
+func zeroStruct<T>(_ ptr: UnsafeMutablePointer<T>) {
```

Comment:
this can likely be used instead of the initialize stuff you had

Reply:
corrected